### PR TITLE
add links to minutes on githut as well as etherpads

### PR DESCRIPTION
**Pull request by fkr**
Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

**Review by itrich — APPROVED** [review]
Looks good! I'll add a seperate issue in the website repo to render the links as clickable href.

File: team_meetings.yml [hunk 1]
```diff
@@ -44,6 +46,8 @@ events:
       This is our bi-weekly meeting for Team IaaS in which we deep-dive into topics.
       There is a github issue to which items that need to be discussed can be added: https://github.com/SovereignCloudStack/issues/issues/46
 
+      Minutes: https://github.com/SovereignCloudStack/minutes/tree/main/iaas
```
Comment (berendt) [inline team_meetings.yml:49]: iaas -> container

File: team_meetings.yml [hunk 2]
```diff
@@ -44,6 +46,8 @@ events:
...
+      Etherpad: https://input.osb-alliance.de/p/2022-scs-team-iaas
```
Comment (berendt) [inline team_meetings.yml:50]: iaas -> container

**Comment by berendt** [comment]
@fkr Links for the container team are wrong. Do you fix them in a followup PR?

File: team_meetings.yml (hunk 2 above)
Comment (berendt) [reply team_meetings.yml:50]: Ignore...

File: team_meetings.yml (hunk 1 above)
Comment (berendt) [reply team_meetings.yml:49]: Ignore...

**Pull request merged**